Mathlib/Algebra/Order/Monoid/OrderDual.lean

```diff
@@ -0,0 +1,98 @@
+/-
+Copyright (c) 2016 Jeremy Avigad. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jeremy Avigad, Leonardo de Moura, Mario Carneiro, Johannes Hölzl
+-/
+import Mathlib.Algebra.Group.OrderSynonym
+import Mathlib.Algebra.Order.Monoid.Cancel.Defs
+
+/-! # Ordered monoid structures on the order dual. -/
+
+
+universe u
+
+variable {α : Type u}
+
+open Function
+
+namespace OrderDual
+
+@[to_additive]
+instance contravariant_class_mul_le [LE α] [Mul α] [c : ContravariantClass α α (· * ·) (· ≤ ·)] :
```

Comment:
```suggestion
instance contravariant_class_mul_le [LE α] [Mul α] [c : ContravariantClass α α (· * ·) (· ≤ ·)] :
```
This should be called `contravariantClass_mul_le`. Could you rename?

You'll also need to add `#align` statements for the additive versions. (Hopefully this can be automated, see https://github.com/leanprover-community/mathport/issues/205.)